src/backend/expungeservice/form_filling.py

```diff
@@ -705,4 +706,5 @@ def counties_with_cases_to_expunge(all_case_results: List[CaseResults]):
             if case_result.case.summary.balance_due_in_cents != 0:
                 counties_with_balances.append(case_result.case.summary.location)
         counties_with_expungements = [county for county in counties_with_eligible_charge if county not in counties_with_balances]
-        return counties_with_expungements
+        return []
```

Comment:
Not a big deal since the result of this function isn't used anymore, but I think this change in particular is confusing since it breaks the functionality. I would either leave it alone or comment out the function.